.github/workflows/platform-checkout-test-site-sync.yml

```
@@ -0,0 +1,48 @@
+# Sync the WCPay plugin to the WooCommerce Platform Checkout test site.
+
+name: WooCommerce Platform Checkout Test Site Auto-Sync
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push events for the develop branch.
+  push:
+    branches:
+      - develop
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build-and-upload"
+  build-and-upload:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout develop branch
+        uses: actions/checkout@v2
+        with:
+          ref: 'develop'
+
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+
```

Also did you consider using the cache option? Seems like a good idea since it's being rebuild with ever commit to develop, right? A cache would probably save some resources.
```suggestion
          cache: 'npm'

```

---

Awesome suggestion! It looks like it uses `actions/cache` under the hood which makes it easy.